v2/robotmk/environment.py

```diff
@@ -31,6 +34,7 @@ def wrap_for_execution(self, command: Sequence[str]) -> list[str]:
             str(self.binary),
             "task",
             "script",
+            *self._controller_and_space_args(),
```

Comment:
OTI: was under the impression that the `-r` argument would automatically select the correct space.

Reply:
I don't think that `-r` has any impact on the space. From my understanding, unless explicitly specified, you always end up with the default (`"user"`).